tests/testthat.R

```diff
@@ -1,19 +1,21 @@
 library(testthat)
 
-KBC_TOKEN <- 'your_token'
-KBC_URL <- 'https://connection.keboola.com/v2/'
-
-# override with config if any
-if (file.exists("config.R")) {
-  source("config.R")
-}
-
-# override with environment if any
-if (nchar(Sys.getenv("KBC_TOKEN")) > 0) {
-  KBC_TOKEN <- Sys.getenv("KBC_TOKEN")  
+var <- Sys.getenv('KBC_TEST_TOKEN')
+if (var == '') {
+    stop('KBC_TEST_TOKEN environment variable is empty')
 }
-if (nchar(Sys.getenv("KBC_URL")) > 0) {
-  KBC_URL <- Sys.getenv("KBC_URL")  
+var <- Sys.getenv('KBC_TEST_URL')
+if (var == '') {
+    stop('KBC_TEST_URL environment variable is empty')
 }
 
 test_check("keboola.sapi.r.client")
+
+#Sys.setenv(KBC_TEST_TOKEN='66-322-DyChVD3tyv6bzasIU8YCC9uCcFztnC5SGjBLqP7p')
+#Sys.setenv(KBC_TEST_URL='https://connection.north-europe.azure.keboola.com/')
```

Comment:
@odinuv probably want to refresh this ;-)

Reply:
done